src/components/badges/Badges.props.ts

```diff
@@ -4,4 +4,8 @@ type PMentionBadge = {
     mentionLimit?: number;
 };
 
-export default PMentionBadge;
+type PUnreadBadge = {
+    width?: number;
+};
+
+export type { PMentionBadge, PUnreadBadge };
```

Comment:
I dont think  mixing 2 components in one folder is a good idea.
When treated as a separate component each of them should have their own files.